tests/test_telescope_loader.py

```diff
@@ -344,3 +344,272 @@ def test_config_zernike_perturbation():
             'telescope'
         )['base']
         assert perturbed == ref
+
+
+def test_config_fea():
+    """Test that we can use batoid_rubin package to add FEA perturbations.
+    """
+
+    # It's difficult to directly add a bending mode or printthrough to a
+    # surface figure, but we can at least test that we can manipulate rigid
+    # body degrees-of-freedom either directly or through the FEA interface.
+
+    # First standard degree of freedom is M2 -dz in microns
+    telescope = imsim.load_telescope("LSST_r.yaml")
+    ref = (telescope
+        .withGloballyShiftedOptic(
+            'M2',
+            [0, 0, 1e-6]
+        )
+    )
+    dof = [-1.0]+[0.0]*49
+    dofstr = ",".join(str(s) for s in dof)
+
+    perturb_config = textwrap.dedent(
+        f"""
+        input:
+            telescope:
+                file_name: LSST_r.yaml
+                fea:
+                    aos_dof:
+                        dof: [{dofstr}]
+        """
+    )
+
+    config = yaml.safe_load(perturb_config)
+    galsim.config.ProcessInput(config)
+    perturbed = galsim.config.GetInputObj(
+        'telescope',
+        config['input']['telescope'],
+        config,
+        'telescope'
+    )['base']
+    assert ref == perturbed
```

Comment:
Is there a more direct way to do this without having to know the specific order of values in the dof list?  I was expected to see a follow up to this one that looked like:
```
 perturb_config = textwrap.dedent(
        f"""
        input:
            telescope:
                file_name: LSST_r.yaml
                fea:
                    m2_position
                        dz: -1.e-6
        """
```
That seems more user-friendly for the kinds of things that e.g. Virginia is working on with making donuts.  Does she always need to have configs with lists of 50 numbers, one of which is non-zero?

Reply:
Yeah, see above comment.  There already is a simpler interface for this type of thing.